xml/depl_multiple_networks.xml

```diff
@@ -0,0 +1,433 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter
+[
+<!ENTITY % entities SYSTEM "entity-decl.ent">
+%entities;
+]>
+
+<chapter xmlns="http://docbook.org/ns/docbook" 
+xmlns:xi="http://www.w3.org/2001/XInclude" 
+xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" 
+xml:id="sec.depl.multi_external_networks">
+ <info>
+  <title>Setting Up Multiple External Networks</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer>cs</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>no</dm:translation>
+   <dm:languages/>
+  </dm:docmanager>
+  <abstract>
+   <para>
+This guide shows you how to set up and test multiple external networks 
+on &cloud;, using the &o_netw; &barcl;, the <command>mkcloud</command> 
+automation tools, and Open vSwitch.
+   </para>
+  </abstract>
+ </info>
+
+ <sect1  xml:id="sec.nettest">
+     <title>Test Network Configuration</title>
+ <para>
+Install &cloud; using &crow; and setting the Open vSwitch plugin for 
+the &o_netw; &barcl;. When the cloud is up, run the following test to confirm 
+that the network configuration is correct. Log in to the node where &o_netw;  
+services are running  (for all these tests we will assume we are using this 
+node).
+ </para>
+ <procedure xml:id="pro.secgroup">
+  <step>
+   <para>
+Create a security group that allows logging in via SSH into your virtual 
+machines (VMs):
+</para>
+<screen>
```

Comment:
I don't see the difference on my local builds. Nevertheless, line breaks begone!